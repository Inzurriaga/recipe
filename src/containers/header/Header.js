import React, { Component } from "react"
import { connect } from "react-redux"

export class Header extends Component {
  render(){
    return(
      <header>
        <div>
          <h1>
            <span>R</span>
            <span>E</span>
            <span>C</span>
            <span>I</span>
            <span>P</span>
            <span>E</span>
          </h1>
        </div>
        <form>
          <input></input>
          <button></button>
        </form>
      </header>
    )
  }
}

export default connect()(Header)