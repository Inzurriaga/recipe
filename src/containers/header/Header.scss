header {
  position: fixed;
  display: flex;
  flex: 1fr 1fr;
  justify-content: space-between;
  width: 100%;
  height: 125px;
  background-color: #f9bf21;
  h1 {
    margin-left: 40px;
    font-family: 'Bevan', cursive;
    text-shadow: 2px 5px 5px #000000;
    color: #ffffff;
    font-size: 4em;
    span {
      padding: 10px 20px;
      margin: 10px;
      background-color: #513a24;
    }
  }
  div {
    display: flex;
    justify-content: center;
    align-items: center;
    width: 100%;
    margin: 10px;
    input {
      font-size: 2em;
      padding-left: 5px;
      height: 48px;
      width: 50%;
      border-radius: 5px 0px 0px 5px;
      outline: none;
      border: none;
    }
    button {
      cursor: pointer;
      background-color: black;
      height: 50px;
      width: 50px;
      border-radius: 0px 5px 5px 0px;
      outline: none;
      border: none;
    }
  }
}