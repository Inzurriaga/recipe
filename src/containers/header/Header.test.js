import React from 'react';
import { Header } from './Header';
import { shallow } from "enzyme";

const mockSetRecipes = jest.fn();

describe("Header", () => {

  describe("Header", () => {
    let wrapper;

    beforeEach(() => {
      wrapper = shallow(
        <Header setRecipes={mockSetRecipes}/>
      );
    })

    it("should match snap shot", () => {
      expect(wrapper).toMatchSnapshot();
    })
  })

  describe("mapDispatchToProps", () => {

  })
})