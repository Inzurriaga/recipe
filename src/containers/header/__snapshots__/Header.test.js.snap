// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header should match snap shot 1`] = `ShallowWrapper {}`;
